### PR TITLE
fix(ocpp): avoid an error log in handling downstream messages

### PR DESCRIPTION
**Pull request by HJianBo**
Small fixes for:
- https://emqx.atlassian.net/browse/EMQX-11620
- https://emqx.atlassian.net/browse/EMQX-11495
- https://emqx.atlassian.net/browse/EMQX-11497

Release version: v/e5.7.0

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- ~Added property-based tests for code which performs user input validation~
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- ~Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket~
- ~Schema changes are backward compatible~

## Checklist for CI (.github/workflows) changes

- ~If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)~
- ~Change log has been added to `changes/` dir for user-facing artifacts update~

### CODE REVIEW

File: apps/emqx_gateway/src/emqx_gateway_api_listeners.erl [hunk 1]
```diff
@@ -762,9 +775,9 @@ examples_listener() ->
                                     <<"tlsv1.1">>,
                                     <<"tlsv1">>
                                 ],
-                                cacertfile => <<"/etc/emqx/certs/cacert.pem">>,
-                                certfile => <<"/etc/emqx/certs/cert.pem">>,
-                                keyfile => <<"/etc/emqx/certs/key.pem">>,
+                                cacertfile => <<"etc/certs/cacert.pem">>,
```
Comment (zmstone) [inline apps/emqx_gateway/src/emqx_gateway_api_listeners.erl:778]:
```suggestion
                                cacertfile => <<"${EMQX_ETC_DIR}/certs/cacert.pem">>,
```

File: changes/ee/fix-12892.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Fix a logical error in OCPP gateway's handling of downstream BootNotification.
```
Comment (zmstone) [inline changes/ee/fix-12892.md:1]:
```suggestion
Fix an error in OCPP gateway's handling of downstream BootNotification.
```

File: changes/ee/fix-12892.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+Fix the `gateways/ocpp/listeners` endpoint to return the correct current connection number.
```
Comment (zmstone) [inline changes/ee/fix-12892.md:3]:
```suggestion
Fix the `gateways/ocpp/listeners` endpoint to return the correct number of current connections.
```

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[Test Execution]()

**Pull request merged**